contributor-advice.md

```diff
@@ -26,18 +35,19 @@ Alpakka, same as Akka, aims to keep 100% feature parity between the various lang
 
 ### Settings
 
+> Reference connector [settings classes](reference/src/main/scala/akka/stream/alpakka/reference/settings.scala)
```

Comment:
lowercase `settings.scala`?

Reply:
Yes, which would mean that there are multiple logical units of compilation - settings classes. This is from https://docs.scala-lang.org/style/files.html

Usually a connector will have `settings.scala` and `model.scala`. The latter is for read/write messages.

Reply:
ok, that's a style I don't use, because I don't like files with many unrelated classes. Often there is one main class, and a few related that can be grouped together and then the file should have the name of the main class.

but I guess it's a matter of taste